examples/heat_equation/source/heat_ida.cc

```diff
@@ -484,19 +492,29 @@ bool Heat<dim>::solver_should_restart (const double t,
                                           0,
                                           0,
                                           triangulation->locally_owned_subdomain());
+
+
       max_kelly = estimated_error_per_cell.linfty_norm();
       max_kelly = Utilities::MPI::sum(max_kelly, comm);
```

Comment:
Well, the `linfty_norm()` should be the `max`, not the sum. 

I'd use 

~~~
max_kelly = Uitlities::MPI::max(max_kelly, comm);
~~~